tests/drivers/uart/uart_mixed_modes/Kconfig

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+config ASYNC_TO_INT
+	bool "Enable async to int mode uart transmission test"
```

Comment:
Kconfigs do not start with "Enable"

Reply:
> Why ? I see quite a lot descriptions starting with "Enable.." in NCS

Where? Seemingly the Kconfig compliance check is not running in tests or samples and needs investigating

Reply:
> Why ? I see quite a lot descriptions starting with "Enable.." in NCS

Because:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
<img width="1155" height="198" alt="image" src="https://github.com/user-attachments/assets/7b9e3750-357d-4130-865c-db5b0a12669d" />